taskt/UI/Forms/frmScreenRecorder.cs

```diff
@@ -1,93 +1,68 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.ComponentModel;
-using System.Data;
-using System.Diagnostics;
-using System.Drawing;
-using System.Linq;
-using System.Runtime.InteropServices;
-using System.Text;
-using System.Threading.Tasks;
-using System.Windows.Forms;
 using taskt.Core.Automation.Commands;
 using taskt.Core.Automation.User32;
+using taskt.Properties;
 
 namespace taskt.UI.Forms
 {
-    public partial class frmSequenceRecorder : UIForm
+    public partial class frmScreenRecorder : UIForm
     {
-
-        public static List<Core.Automation.Commands.ScriptCommand> commandList;
         public frmScriptBuilder callBackForm { get; set; }
```

Comment:
Please use PascalCase for public properties.

Reply:
done